doc/book/replication/repl_leader_elect.rst

```diff
@@ -60,46 +61,45 @@ for itself and sends vote requests to other nodes.
 Upon receiving vote requests, a node votes for the first of them, and then cannot
 do anything in the same term but wait for a leader being elected.
 
-A node that collected a :ref:`quorum of votes <repl_leader_elect_config>`
-becomes a leader
-and notifies other nodes about that. Also a split-vote can happen
+The node that collected a :ref:`quorum of votes <repl_leader_elect_config>`
+becomes the leader
+and notifies other nodes about that. Also, a split vote can happen
 when no nodes received a quorum of votes. In this case,
 after a :ref:`random timeout <repl_leader_elect_config>`,
 each node increases its term and starts a new election round if no new vote
-request with greater term is arrived during this time period.
-Eventually a leader is elected.
+request with a greater term arrives during this time period.
+Eventually, a leader is elected.
 
 All the non-leader nodes are called *followers*. The nodes that start a new
 election round are called *candidates*. The elected leader sends heartbeats to
 the non-leader nodes to let them know it is alive. So if there are no heartbeats
-for a time period set by the :ref:`replication_timeout <cfg_replication-replication_timeout>`
-option, a new election is started. Terms and votes are persisted by
+for a period set by the :ref:`replication_timeout <cfg_replication-replication_timeout>`
+option, a new election starts. Terms and votes are persisted by
 each instance in order to preserve certain Raft guarantees.
 
 During the election, the nodes prefer to vote for those ones that have the
 newest data. So as if an old leader managed to send something before its death
 to a quorum of replicas, that data wouldn't be lost.
 
-When :ref:`election is enabled <repl_leader_elect_config>`, it is required
-to have connections
-between each node pair so as it would be the fullmesh topology. This is needed
+When :ref:`election is enabled <repl_leader_elect_config>`, there must be connections
+between each node pair so as it would be the full-mesh topology. This is needed
 because election messages for voting and other internal things need direct
 connection between the nodes.
 
-Also if election is enabled on the node, it won't replicate from any nodes except
+Also, if an election is enabled on the node, it won't replicate from any nodes except
```

Comment:
I think it should be without the 'an' article in this case -- just "... if election is enabled..." -- because this is not about an election instance ('an election') but about the settings, the general possibility for a node to take part in the election process.